include/wx/defs.h

```diff
@@ -3042,6 +3039,26 @@ typedef const void* WXWidget;
 #define DECLARE_NO_ASSIGN_CLASS(classname) \
     wxDECLARE_NO_ASSIGN_CLASS(classname);
 
+/*  --------------------------------------------------------------------------- */
+/*  macros to define a class with default copy constructor and assignment operator
+/*  --------------------------------------------------------------------------- */
```

Comment:
This comment results in a ton of warning in Travis builds, it should be rewritten to either use single `/* ... */` around it or add missing `*/` to the second line.